src/filter.js

```diff
@@ -223,19 +213,6 @@ define(function (require) {
    *  @method connect
    *  @param  {Object} unit
    */
-  p5.Filter.prototype.connect = function(unit) {
-    var u = unit || p5.soundOut.input;
-    this.output.connect(u);
-  };
-
-  /**
-   *  Disconnect all output.
-   *  
-   *  @method disconnect
-   */
-  p5.Filter.prototype.disconnect = function() {
-    this.output.disconnect();
-  };
 
   p5.Filter.prototype.dispose = function() {
     // remove reference from soundArray
```

Comment:
p5.Filter's dispose method can call `p5.Effect.prototype.dispose.apply(this)` and then just dispose of the pieces that are unique to this class.

For methods that extend superclass methods and have arguments, you can also pass in the `arguments` object like this:

`p5.Effect.prototype.method.apply(this, arguments)`